hw-1/output.txt

```diff
@@ -0,0 +1,5 @@
+{"book":"The Compound Effect","author":"Darren Hardy","amount":"5","price":"9.48"}
```

Comment:
You need to commit this file with no content

Reply:
Done